docs/11-circleci/03-build.mdx

```diff
@@ -285,6 +285,27 @@ workflows:
 - _Required_: `false`
 - _Type_: `boolean`
 
+#### fetch-submodules
+
+Whether to fetch git submodules relating to the project repository
```

Comment:
It is just missing the period (.) at the end. Otherwise looks good!

```diff
-Whether to fetch git submodules relating to the project repository
+Whether to fetch git submodules relating to the project repository.
```